appeals/e2e/cypress/e2e/back-office-appeals/changeStartDate.spec.js

```diff
@@ -0,0 +1,30 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { happyPathHelper } from '../../support/happyPathHelper';
+import { ListCasesPage } from '../../page_objects/listCasesPage';
+import { DateTimeSection } from '../../page_objects/dateTimeSection';
+
+const listCasesPage = new ListCasesPage();
+const dateTimeSection = new DateTimeSection();
+const caseDetailsPage = new CaseDetailsPage();
+//const date = new Date(); // Note: Months are 0-indexed, so 10 is November
```

Comment:
Does this need to be here? Appears to be left over comment from working on `verifyChangeStartDate` method

Reply:
I can't see this on my local code, maybe I deleted it but didn't commit? It's not appearing in my change list though?